vhdbuilder/packer/packer-plugin.pkr.hcl

```diff
@@ -0,0 +1,8 @@
+packer {
+  required_plugins {
+    azure = {
+      version = ">= 1.4.3"
```

Comment:
can we specify a version here to avoid any possible issue in the latest version?